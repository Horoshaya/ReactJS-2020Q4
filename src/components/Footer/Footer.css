.wrapper {
  padding: 30px 10px;
  background: #424242;
}
