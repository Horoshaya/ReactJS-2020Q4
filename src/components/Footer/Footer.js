import React from "react";

const Footer = () => {
  return (
    <h2>
      <span>NETFLIX</span>roulette
    </h2>
  );
};

export default Footer;
