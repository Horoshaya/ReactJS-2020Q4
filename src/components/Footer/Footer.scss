.wrapper {
  padding: 30px 10px;
  background: #424242;
  color: #f65261;
  font-family: "Trebuchet MS", "Lucida Sans Unicode", "Lucida Grande",
    "Lucida Sans", Arial, sans-serif;
}

.title {
  text-align: center;
  font-weight: 100;
}

.titleAccent {
  font-weight: 900;
}
