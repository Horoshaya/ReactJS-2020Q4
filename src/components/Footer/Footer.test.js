import React from 'react';
import Footer from './Footer';
import renderer from 'react-test-renderer';

test('FooterSection matches the Snapshot', () => {
  const component = renderer.create(<Footer />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
