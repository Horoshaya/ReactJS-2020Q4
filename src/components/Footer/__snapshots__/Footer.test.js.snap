// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterSection matches the Snapshot 1`] = `
<footer>
  <h2>
    <span>
      netflix
    </span>
    roulette
  </h2>
</footer>
`;
